packages/web-features-figuring/src/browser-compat-data/release.ts

```diff
@@ -0,0 +1,54 @@
+import { Temporal } from "@js-temporal/polyfill";
+import { ReleaseStatement } from "@mdn/browser-compat-data";
+
+import { Browser } from "./browser";
+
+export class Release {
```

Comment:
I like this helper! A release as a tuple of browser and version string is very convenient, and I've written some pretty inconvenient code passing around the two as a pair.